templates/pioneer-study/example.1.schema.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
the path should be  templates/pioneer-study/example/example.1.schema.json

You will also want a validation/pioneer-study/example.1.sample.json containing
```json
{ "exampleString" : "foobar" }
```